app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -32,16 +44,19 @@ class NewAction extends \Magento\Newsletter\Controller\Subscriber
      * @param StoreManagerInterface $storeManager
      * @param CustomerUrl $customerUrl
      * @param CustomerAccountManagement $customerAccountManagement
+     * @param EmailAddress $emailValidator
      */
     public function __construct(
         Context $context,
         SubscriberFactory $subscriberFactory,
         Session $customerSession,
         StoreManagerInterface $storeManager,
         CustomerUrl $customerUrl,
-        CustomerAccountManagement $customerAccountManagement
+        CustomerAccountManagement $customerAccountManagement,
+        EmailAddress $emailValidator
```

Comment:
According to https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/ 
 Please make this parameter optional and get object from ObjectManager as fallback.